sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -79,37 +79,59 @@ export class ServiceBusClient {
   }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in peekLock mode.
+   *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
+   * queue.
+   *
+   * Messages that are not settled will be redelivered as many times as the max delivery count
+   * set on the queue.
+   *
+   * You can settle a message by calling complete(), abandon(), defer() or deadletter() methods on
+   * the message.
```

Comment:
If we go with the suggestion in https://github.com/Azure/azure-sdk-for-js/pull/8778/files#r421840508, then the above should go right after, and then followed by what happens when you don't settle

Reply:
Merged the idea.